pkgs/development/libraries/gsl/default.nix

```diff
@@ -9,10 +9,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-3LD71DBIgyt1f/mUJpGo3XACbV2g/4VgHlJof23us0s=";
   };
 
+  outputs = [ "out" "dev" ];
```

Comment:
Most common convention is placing outputs above `src`, see https://github.com/jtojnar/nixpkgs-hammering/blob/0c54b7121a21f79e580773a46fc8d51482dc1d1c/explanations/attribute-ordering.md.